templates/modules/b2b-account/account-hierarchy/parent-b2baccount.hypr.live

```diff
@@ -0,0 +1,44 @@
+{% require_script "modules/b2b-account/parent-b2baccount" %}
+
+{% extends "modules/common/modal-dialog" %}
+<div class="modal-header">
+  {% block modal-header %}
+  <button type="button" class="close" aria-hidden="true"  data-dismiss="modal" data-mz-action="handleDialogCancel">×</button>
+  <h1 class="modal-title" style="text-align: center;">Change Parent Account</h1>
```

Comment:
remove inline styling in this template and add a CSS, classes

Reply:
done